src/app/components/profile/profile.html

```diff
@@ -83,7 +83,7 @@
 
     <!-- PRIVATE INFO -->
 
-    <div class="col-md-8 col-md-offset-2 col-lg-6 col-lg-offset-3 col-sm-12" ng-hide="true">
+    <div class="col-md-8 col-md-offset-2 col-lg-6 col-lg-offset-3 col-sm-12" <!-- ng-hide="true" -->>
```

Comment:
There is no information available to show, must have forgot to undo the changes made